.github/workflows/backward-compatibility.yml

```diff
@@ -0,0 +1,40 @@
+name: Build Against Previous Versisons of Kotlin
+on:
+  push:
+    branches:
+      - master
+      - "3.0"
```

Comment:
I don't think we have a 3.0 branch

Reply:
This is a copy/paste from the other workflow. Want me to remove it?

Reply:
Ack, yes please